File: Data Structures & Algorithms/is-palindrome/submission-0.java
class Solution {
    public boolean isPalindrome(String s) {
        boolean pal = true;
        String word = s.toLowerCase().replaceAll("[^A-Za-z0-9]", "");
        System.out.println(word);
        for(int i = 0; i < word.length() / 2; i++)
        {
            if(word.charAt(i) != word.charAt(word.length() - 1 - i))
            {
                pal = false;
                break;
            }
        }
        return pal;
    }
}

File: Data Structures & Algorithms/word-break/submission-0.py
class Solution:
    def wordBreak(self, s: str, wordDict: List[str]) -> bool:
        dp = [False] * (len(s)+1)
        dp[0] = True
        for i in range(1, len(s)+1):
            for word in wordDict:
                if i-len(word) >= 0 and s[i-len(word):i] == word:
                    dp[i] = dp[i-len(word)]
                    if dp[i]:
                        break
        print(dp)
        return dp[len(s)]